lib/fiddle/truffleruby.rb

```diff
@@ -0,0 +1,623 @@
+# frozen_string_literal: true
+# truffleruby_primitives: true
+
+# Copyright (c) 2019, 2024 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 2.0, or
+# GNU General Public License version 2, or
+# GNU Lesser General Public License version 2.1.
```

Comment:
We will need to relicense this code to upstream here (otherwise we would have to change the fiddle gem license).
From `git shortlog -sne lib/truffle/truffle/fiddle_backend.rb` it looks like it was only modified by Oracle employees, so then we should be able to relicense that, by relicensing https://github.com/oracle/truffleruby/blob/master/lib/truffle/truffle/fiddle_backend.rb under BSD-2-Clause.

Reply:
Thanks. It's very helpful.